### ISSUE TITLE
[Bug]: generated image is all grey

### ISSUE THREAD

**Issue by Surya-733** [lshqqytiger/stable-diffusion-webui-directml#661]
### Checklist

- [x] The issue exists after disabling all extensions
- [x] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [x] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

i just install the sd, load the model i download, then wait for compiling for awhile, then i try to generate image with text to img its just give me grey result

### Steps to reproduce the problem

1. im using rx 6700
2. follow this installation instruction "https://github.com/CS1o/Stable-Diffusion-Info/wiki/Webui-Installation-Guides#amd-automatic1111-with-zluda"
3. load a downloaded model
4. try to generate
5. the result is all grey

### What should have happened?

generate a proper image

### Sysinfo

{
    "Platform": "Windows-10-10.0.26200-SP0",
    "Python": "3.11.9",
    "Version": "v1.10.1-amd-56-g9cb6e4c5",
    "Commit": "9cb6e4c5431b440b55dc5da037bcdbcf220bf511",
    "Git status": "On branch master\nYour branch is up to date with 'origin/master'.\n\nChanges not staged for commit:\n  (use \"git add <file>...\" to update what will be committed)\n  (use \"git restore <file>...\" to discard changes in working directory)\n\tmodified:   webui-user.bat\n\nUntracked files:\n  (use \"git add <file>...\" to include in what will be committed)\n\t.zluda/\n\nno changes added to commit (use \"git add\" and/or \"git commit -a\")",
    "Script path": "F:\\SD-Zluda\\stable-diffusion-webui-amdgpu",
    "Data path": "F:\\SD-Zluda\\stable-diffusion-webui-amdgpu",
    "Extensions dir": "F:\\SD-Zluda\\stable-diffusion-webui-amdgpu\\extensions",
    "Checksum": "4b8972987b60679f757a4e9e76bc42a70411ea2b13b9db728dd5a7008aff3507",
    "Commandline": [
        "launch.py",
        "--use-zluda",
        "--update-check",
        "--skip-ort",
        "--skip-python-version-check"
    ],
    "Torch env info": {
        "torch_version": "2.7.1+cu118",
        "is_debug_build": "False",
        "cuda_compiled_version": "11.8",
        "gcc_version": null,
        "clang_version": "19.0.0git (git@github.amd.com:Compute-Mirrors/llvm-project 5353ca3e0e5ae54a31eeebe223da212fa405567a)",
        "cmake_version": null,
        "os": "Microsoft Windows 11 Home (10.0.26200 64-bit)",
        "libc_version": "N/A",
        "python_version": "3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)] (64-bit runtime)",
        "python_platform": "Windows-10-10.0.26200-SP0",
        "is_cuda_available": "True",
        "cuda_runtime_version": null,
        "cuda_module_loading": "LAZY",
        "nvidia_driver_version": null,
        "nvidia_gpu_models": null,
        "cudnn_version": null,
        "pip_version": "pip3",
        "pip_packages": [
            "numpy==1.26.4",
            "onnx==1.16.2",
            "onnx-ir==1.0.0",
            "onnxruntime==1.26.0",
            "onnxscript==0.5.7",
            "open-clip-torch==2.20.0",
            "pytorch-lightning==1.9.4",
            "torch==2.7.1+cu118",
            "torchdiffeq==0.2.3",
            "torchmetrics==1.9.0",
            "torchsde==0.2.6",
            "torchvision==0.22.1+cu118"
        ],
        "conda_packages": null,
        "hip_compiled_version": "N/A",
        "hip_runtime_version": "N/A",
        "miopen_runtime_version": "N/A",
        "caching_allocator_config": "",
        "is_xnnpack_available": "True",
        "cpu_info": [
            "Name: AMD Ryzen 5 8600G w/ Radeon 760M Graphics      ",
            "Manufacturer: AuthenticAMD",
            "Family: 107",
            "Architecture: 9",
            "ProcessorType: 3",
            "DeviceID: CPU0",
            "CurrentClockSpeed: 4351",
            "MaxClockSpeed: 4351",
            "L2CacheSize: 6144",
            "L2CacheSpeed: None",
            "Revision: 29954"
        ]
    },
    "Exceptions": [],
    "CPU": {
        "model": "AMD64 Family 25 Model 117 Stepping 2, AuthenticAMD",
        "count logical": 12,
        "count physical": 6
    },
    "RAM": {
        "total": "32GB",
        "used": "11GB",
        "free": "20GB"
    },
    "Extensions": [],
    "Inactive extensions": [],
    "Environment": {
        "COMMANDLINE_ARGS": "--use-zluda --update-check --skip-ort --skip-python-version-check",
        "GRADIO_ANALYTICS_ENABLED": "False"
    },
    "Config": {
        "ldsr_steps": 100,
        "ldsr_cached": false,
        "SCUNET_tile": 256,
        "SCUNET_tile_overlap": 8,
        "SWIN_tile": 192,
        "SWIN_tile_overlap": 8,
        "SWIN_torch_compile": false,
        "hypertile_enable_unet": false,
        "hypertile_enable_unet_secondpass": false,
        "hypertile_max_depth_unet": 3,
        "hypertile_max_tile_unet": 256,
        "hypertile_swap_size_unet": 3,
        "hypertile_enable_vae": false,
        "hypertile_max_depth_vae": 3,
        "hypertile_max_tile_vae": 128,
        "hypertile_swap_size_vae": 3,
        "sd_model_checkpoint": "vixonSMilkFactory_animaV2.safetensors [d9ebc91527]",
        "sd_checkpoint_hash": "d9ebc91527e7ef33048f88d57168500d376c379347546ec4963631ffa0fb5eae"
    },
    "Startup": {
        "total": 12.943347215652466,
        "records": {
            "initial startup": 0.02362060546875,
            "prepare environment/checks": 0.0029993057250976562,
            "prepare environment/git version info": 0.08911299705505371,
            "prepare environment/clone repositores": 0.5720808506011963,
            "prepare environment/run extensions installers": 0.0,
            "prepare environment/check version": 0.686112642288208,
            "prepare environment": 15.534397840499878,
            "launcher": 0.0024335384368896484,
            "import torch": 0.0,
            "import gradio": 0.0,
            "setup paths": 0.0007271766662597656,
            "import ldm": 0.002021312713623047,
            "import sgm": 0.0,
            "initialize shared": 0.4277205467224121,
            "other imports": 0.2915167808532715,
            "opts onchange": 0.0,
            "setup SD model": 0.0,
            "setup codeformer": 0.0020020008087158203,
            "setup gfpgan": 0.010791301727294922,
            "set samplers": 0.0,
            "list extensions": 0.0009996891021728516,
            "restore config state file": 0.0,
            "list SD models": 0.2843804359436035,
            "list localizations": 0.0006482601165771484,
            "load scripts/custom_code.py": 0.006071805953979492,
            "load scripts/img2imgalt.py": 0.00099945068359375,
            "load scripts/loopback.py": 0.0,
            "load scripts/outpainting_mk_2.py": 0.004758358001708984,
            "load scripts/poor_mans_outpainting.py": 0.0010120868682861328,
            "load scripts/postprocessing_codeformer.py": 0.0004756450653076172,
            "load scripts/postprocessing_gfpgan.py": 0.0,
            "load scripts/postprocessing_upscale.py": 0.0040056705474853516,
            "load scripts/prompt_matrix.py": 0.001001596450805664,
            "load scripts/prompts_from_file.py": 0.0007777214050292969,
            "load scripts/sd_upscale.py": 0.000476837158203125,
            "load scripts/xyz_grid.py": 0.005024909973144531,
            "load scripts/ldsr_model.py": 0.5075979232788086,
            "load scripts/lora_script.py": 0.28634119033813477,
            "load scripts/scunet_model.py": 0.10169076919555664,
            "load scripts/swinir_model.py": 0.17007780075073242,
            "load scripts/hotkey_config.py": 0.1351478099822998,
            "load scripts/extra_options_section.py": 0.01203012466430664,
            "load scripts/hypertile_script.py": 0.04046797752380371,
            "load scripts/postprocessing_autosized_crop.py": 0.10707950592041016,
            "load scripts/postprocessing_caption.py": 0.0,
            "load scripts/postprocessing_create_flipped_copies.py": 0.0009975433349609375,
            "load scripts/postprocessing_focal_crop.py": 0.00602412223815918,
            "load scripts/postprocessing_split_oversized.py": 0.0010008811950683594,
            "load scripts/soft_inpainting.py": 0.003996849060058594,
            "load scripts/comments.py": 0.025015592575073242,
            "load scripts/refiner.py": 0.1119999885559082,
            "load scripts/sampler.py": 0.002119779586791992,
            "load scripts/seed.py": 0.0043239593505859375,
            "load scripts": 1.5405158996582031,
            "load upscalers": 0.009234428405761719,
            "refresh VAE": 0.003320455551147461,
            "refresh textual inversion templates": 0.0,
            "scripts list_optimizers": 0.003000020980834961,
            "scripts list_unets": 0.0,
            "reload hypernetworks": 0.0010008811950683594,
            "initialize extra networks": 0.22516369819641113,
            "scripts before_ui_callback": 0.03794074058532715,
            "create ui": 1.223358392715454,
            "gradio launch": 0.4103732109069824,
            "add APIs": 0.01210927963256836,
            "app_started_callback/lora_script.py": 0.0,
            "app_started_callback": 0.0
        }
    },
    "Packages": [
        "accelerate==0.21.0",
        "aenum==3.1.17",
        "aiofiles==23.2.1",
        "aiohappyeyeballs==2.7.1",
        "aiohttp==3.14.3",
        "aiosignal==1.4.0",
        "alembic==1.19.1",
        "altair==5.5.0",
        "antlr4-python3-runtime==4.9.3",
        "anyio==3.7.1",
        "attrs==26.1.0",
        "blendmodes==2022",
        "certifi==2026.7.22",
        "charset-normalizer==3.5.1",
        "clean-fid==0.1.35",
        "click==8.4.2",
        "clip @ https://github.com/openai/CLIP/archive/d50d76daa670286dd6cacf3bcd80b5e4823fc8e1.zip#sha256=b5842c25da441d6c581b53a5c60e0c2127ebafe0f746f8e15561a006c6c3be6a",
        "colorama==0.4.6",
        "colorlog==6.12.0",
        "contourpy==1.3.3",
        "cycler==0.12.1",
        "deprecation==2.1.0",
        "diffusers @ git+https://github.com/huggingface/diffusers@051c8a1c0f5c393a447bef18081fdf94c2a3ab9e",
        "diskcache==5.6.3",
        "einops==0.4.1",
        "facexlib==0.3.0",
        "fastapi==0.94.0",
        "ffmpy==1.0.0",
        "filelock==3.32.3",
        "filterpy==1.4.5",
        "flatbuffers==25.12.19",
        "fonttools==4.63.0",
        "frozenlist==1.8.0",
        "fsspec==2026.7.0",
        "ftfy==6.3.1",
        "gitdb==4.0.12",
        "GitPython==3.1.32",
        "gradio==3.41.2",
        "gradio_client==0.5.0",
        "greenlet==3.5.5",
        "h11==0.12.0",
        "hf-xet==1.6.0",
        "httpcore==0.15.0",
        "httpx==0.24.1",
        "huggingface_hub==0.36.2",
        "idna==3.19",
        "ImageIO==2.37.4",
        "importlib_metadata==9.0.0",
        "importlib_resources==6.5.2",
        "inflection==0.5.1",
        "Jinja2==3.1.6",
        "jsonmerge==1.8.0",
        "jsonschema==4.26.0",
        "jsonschema-specifications==2025.9.1",
        "kiwisolver==1.5.0",
        "kornia==0.6.7",
        "lark==1.1.2",
        "lazy-loader==0.5",
        "lightning-utilities==0.15.3",
        "llvmlite==0.49.0",
        "Mako==1.4.1",
        "MarkupSafe==2.1.5",
        "matplotlib==3.11.1",
        "ml_dtypes==0.5.4",
        "mpmath==1.3.0",
        "multidict==6.7.1",
        "narwhals==2.24.0",
        "networkx==3.6.1",
        "numba==0.67.0",
        "numpy==1.26.4",
        "olive-ai==0.11.0",
        "omegaconf==2.2.3",
        "onnx==1.16.2",
        "onnx-ir==1.0.0",
        "onnxruntime==1.26.0",
        "onnxscript==0.5.7",
        "open-clip-torch==2.20.0",
        "opencv-python==4.11.0.86",
        "optimum==1.25.3",
        "optuna==4.9.0",
        "orjson==3.12.0",
        "packaging==26.3",
        "pandas==2.3.3",
        "piexif==1.1.3",
        "Pillow==9.5.0",
        "pillow-avif-plugin==1.4.3",
        "pip==26.2.1",
        "propcache==0.5.2",
        "protobuf==3.20.2",
        "psutil==5.9.5",
        "pydantic==1.10.26",
        "pydub==0.25.1",
        "pyparsing==3.3.2",
        "python-dateutil==2.9.0.post0",
        "python-multipart==0.0.32",
        "pytorch-lightning==1.9.4",
        "pytz==2026.3.post1",
        "PyWavelets==1.9.0",
        "PyYAML==6.0.3",
        "referencing==0.37.0",
        "regex==2026.7.19",
        "requests==2.34.2",
        "resize-right==0.0.2",
        "rpds-py==2026.6.3",
        "safetensors==0.6.2",
        "scikit-image==0.21.0",
        "scipy==1.17.1",
        "semantic-version==2.10.0",
        "sentencepiece==0.2.2",
        "setuptools==69.5.1",
        "six==1.17.0",
        "smmap==5.0.3",
        "sniffio==1.3.1",
        "spandrel==0.3.4",
        "spandrel_extra_arches==0.1.1",
        "SQLAlchemy==2.0.52",
        "starlette==0.26.1",
        "sympy==1.14.0",
        "tifffile==2026.3.3",
        "timm==1.0.28",
        "tokenizers==0.22.2",
        "tomesd==0.1.3",
        "torch==2.7.1+cu118",
        "torchdiffeq==0.2.3",
        "torchmetrics==1.9.0",
        "torchsde==0.2.6",
        "torchvision==0.22.1+cu118",
        "tqdm==4.70.0",
        "trampoline==0.1.2",
        "transformers==4.57.1",
        "typing_extensions==4.16.0",
        "tzdata==2026.3",
        "urllib3==2.7.0",
        "uvicorn==0.52.3",
        "wcwidth==0.8.2",
        "websockets==11.0.3",
        "wheel==0.48.0",
        "yarl==1.24.5",
        "zipp==4.1.0"
    ]
}

### Console logs

```Shell
venv "F:\SD-Zluda\stable-diffusion-webui-amdgpu\venv\Scripts\Python.exe"
WARNING: ZLUDA works best with SD.Next. Please consider migrating to SD.Next.
Python 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)]
Version: v1.10.1-amd-56-g9cb6e4c5
Commit hash: 9cb6e4c5431b440b55dc5da037bcdbcf220bf511
ROCm: agents=['gfx1031']
ROCm: version=6.2, using agent gfx1031
ZLUDA support: experimental
ZLUDA load: path='F:\SD-Zluda\stable-diffusion-webui-amdgpu\.zluda' nightly=False
Skipping onnxruntime installation.
version check failed 'commit'
W0819 16:42:11.047000 19336 venv\Lib\site-packages\torch\distributed\elastic\multiprocessing\redirects.py:29] NOTE: Redirects are currently not supported in Windows or MacOs.
F:\SD-Zluda\stable-diffusion-webui-amdgpu\venv\Lib\site-packages\timm\models\layers\__init__.py:49: FutureWarning: Importing from timm.models.layers is deprecated, please import via timm.layers
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.layers", FutureWarning)
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
F:\SD-Zluda\stable-diffusion-webui-amdgpu\venv\Lib\site-packages\pytorch_lightning\utilities\distributed.py:258: LightningDeprecationWarning: `pytorch_lightning.utilities.distributed.rank_zero_only` has been deprecated in v1.8.1 and will be removed in v2.0.0. You can import it from `pytorch_lightning.utilities` instead.
  rank_zero_deprecation(
Launching Web UI with arguments: --use-zluda --update-check --skip-ort --skip-python-version-check
Loading weights [d9ebc91527] from F:\SD-Zluda\stable-diffusion-webui-amdgpu\models\Stable-diffusion\vixonSMilkFactory_animaV2.safetensors
Creating model from config: F:\SD-Zluda\stable-diffusion-webui-amdgpu\configs\v1-inference.yaml
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 12.9s (prepare environment: 15.5s, initialize shared: 0.4s, other imports: 0.3s, list SD models: 0.3s, load scripts: 1.5s, initialize extra networks: 0.2s, create ui: 1.2s, gradio launch: 0.4s).
Applying attention optimization: Doggettx... done.
Model loaded in 8.5s (load weights from disk: 0.5s, load config: 0.2s, create model: 3.4s, apply weights to model: 0.5s, apply half(): 1.0s, move model to device: 0.1s, load textual inversion embeddings: 0.4s, calculate empty prompt: 2.5s).
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:03<00:00,  5.62it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:10<00:00,  1.96it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:01<00:00, 25.70it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 30/30 [00:15<00:00,  1.98it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:02<00:00, 13.82it/s]
Total progress:  97%|███████████████████████████████████████████████████████████████▊  | 29/30 [00:01<00:00, 26.16it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 30/30 [00:12<00:00, > 26.16it/s]
```

### Additional information

im using rx 6700, model is vixonSMilkFactory_animaV2.safetensors [d9ebc91527] i downloaded from civitai

**Comment by Speedfist** [comment]
have you tried a different model? Anima is designed specifically for comfyui I think
